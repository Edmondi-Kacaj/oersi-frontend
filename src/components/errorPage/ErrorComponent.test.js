import React from "react";
import { mount, shallow } from "../../enzymeSetUp";
import ErrorComponent from "./ErrorComponent";

describe("ErrorPageComponent ==> UI  ", () => {
  let wrapperMount, wrapperShadow;
  beforeEach(() => {
    wrapperMount = mount(<ErrorComponent />);
    wrapperShadow = shallow(<ErrorComponent />);
  });

  it("MultiListComponent : should render correctly", () => {
    expect(wrapperShadow).toMatchSnapshot();
  });

  it(" ErrorPageComponent:  wraps content in a div with .info class", () => {
    expect(wrapperMount.find(".info").length).toBe(1);
  });

  it(" ErrorPageComponent:  Find Text inside Component", () => {
    expect(
      wrapperShadow
        .text()
        .includes("Sorry something was wrong and we can't load the page")
    ).toBe(true);
  });
});
