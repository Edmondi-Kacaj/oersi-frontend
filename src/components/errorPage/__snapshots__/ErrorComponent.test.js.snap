// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPageComponent ==> UI   MultiListComponent : should render correctly 1`] = `ShallowWrapper {}`;
